crates/core/tedge_actors/src/examples/calculator.rs

```diff
@@ -1,13 +1,12 @@
-use crate::Sender;
-// TODO: make examples respond to RuntimeRequests
+//! Example: implementing a Calculator as an [Actor]
 use crate::message_boxes::MessageReceiver;
 use crate::Actor;
 use crate::RuntimeError;
-use crate::Server;
+use crate::Sender;
 use crate::SimpleMessageBox;
 use async_trait::async_trait;
 
-/// State of the calculator service
+/// State of the calculator actor
 pub struct Calculator {
     state: i64,
```

Comment:
```suggestion
pub struct CalculatorActor {
    state: Calculator,
```
To avoid conflict/confusion with the `Calculator` struct in `calculator_server.rs`.